src/gpu/gpu_reorder_pd.hpp

```diff
@@ -62,9 +69,99 @@ struct gpu_reorder_pd_t : public reorder_pd_t {
                         && post_ops.entry_[0].kind == primitive_kind::sum);
     }
 
-    bool extra_ok() const {
-        return src_md()->extra.flags == 0 && dst_md()->extra.flags == 0;
+    bool extra_ok(bool accept_conv_asymm = false) const {
+        using namespace memory_extra_flags;
+        if (!accept_conv_asymm)
+            return (src_md()->extra.flags == none)
+                    && (dst_md()->extra.flags == none);
+        const uint64_t c = compensation_gpu_conv_asymmetric_src;
+        const uint64_t b = compensation_gpu_conv_asymmetric_src_bwd;
+        const uint64_t s = compensation_gpu_conv_asymmetric_src_swap;
+        return utils::one_of(src_md()->extra.flags, c, c | b, c | b | s, none)
+                && utils::one_of(
+                        dst_md()->extra.flags, c, c | b, c | b | s, none);
     }
+
+    status_t maybe_build_zp_precompute_conv_pd(impl::engine_t *dst_engine) {
+        memory_desc_wrapper dst_mdw(dst_md());
+        auto &extra = dst_mdw.extra();
+        auto needs_conv
+                = memory_extra_flags::compensation_gpu_conv_asymmetric_src;
+        auto is_dst_gpu = (dst_engine->kind() == engine_kind::gpu);
+        do_zp_precomp_conv_ = is_dst_gpu && (extra.flags & needs_conv);
+        if (do_zp_precomp_conv_) {
```

Comment:
To save some indentation.
```suggestion
        if (!do_zp_precomp_conv_) return status::success;
```

Reply:
Done.